services/ui-service/templates/index.html

```diff
@@ -121,25 +121,26 @@
             margin-left: calc(50% - 30px);
             margin-bottom: 20%;
 
-            --d:150px;
+            --d: 150px;
             width: 60px;
             height: 60px;
             border-radius: 100%;
             color: darkred;
-            box-shadow:
-                    calc(1*var(--d))      calc(0*var(--d))     0 0,
-                    calc(0.707*var(--d))  calc(0.707*var(--d)) 0 2px,
-                    calc(0*var(--d))      calc(1*var(--d))     0 4px,
-                    calc(-0.707*var(--d)) calc(0.707*var(--d)) 0 6px,
-                    calc(-1*var(--d))     calc(0*var(--d))     0 8px,
-                    calc(-0.707*var(--d)) calc(-0.707*var(--d))0 10px,
-                    calc(0*var(--d))      calc(-1*var(--d))    0 12px;
+            box-shadow: calc(1 * var(--d)) calc(0 * var(--d)) 0 0,
+            calc(0.707 * var(--d)) calc(0.707 * var(--d)) 0 2px,
+            calc(0 * var(--d)) calc(1 * var(--d)) 0 4px,
+            calc(-0.707 * var(--d)) calc(0.707 * var(--d)) 0 6px,
+            calc(-1 * var(--d)) calc(0 * var(--d)) 0 8px,
+            calc(-0.707 * var(--d)) calc(-0.707 * var(--d)) 0 10px,
+            calc(0 * var(--d)) calc(-1 * var(--d)) 0 12px;
```

Comment:
how do you make sense of this?

Reply:
I've used this tool: https://www.cssportal.com/css-loader-generator/